pkgs/by-name/ks/ksoloti/package.nix

```diff
@@ -0,0 +1,181 @@
+{ ant
+, copyDesktopItems
+, dfu-util
+, fetchpatch
+, fetchurl
+, fetchFromGitHub
+, gcc-arm-embedded-6
+, lib
+, makeDesktopItem
+, makeWrapper
+, stdenv
+, pkg-config
+, systemd
+, unzip
+, zulu
+,
+}:
+let
+  ksolotiVersion = "1.0.12-8";
+  libusbVersion = "1.0.19";
+  chibiOSVersion = "2.6.9";
+
+  src = fetchFromGitHub {
+    owner = "ksoloti";
+    repo = "ksoloti";
+    rev = ksolotiVersion;
+    sha256 = "sha256-UYxCnEZORpfhaHSl6YohuFiQLPoQ1oyJUL9W6Maf8AE=";
+  };
+
+  libusb = stdenv.mkDerivation rec {
```

Comment:
Just for clarity, is there a reason you're using libusb `1.0.19`, e.g incompatability? We already have `1.0.27` packaged under `pkgs.libusb1`.

Reply:
yeah, apparently the patch fails with 1.0.27. I just tried.

Reply:
Hmm, maybe you could try using `libusb-compat-0_1`?